kcidb/orm/argparse.py

```diff
@@ -0,0 +1,93 @@
+"""KCIDB object-relational mapping (ORM)-management
+   command-line argument parsing
+ """
```

Comment:
Please remove the leading whitespace in the last line and format the docstring similar to the others. I.e.:
```suggestion
"""
KCIDB object-relational mapping (ORM)-management
command-line argument parsing
"""
```